zeroex/orderwatch/decoder/event_decoder.go

```diff
@@ -916,6 +914,9 @@ func (d *Decoder) decodeExchange(log types.Log, decodedLog interface{}) error {
 func unpackLog(decodedEvent interface{}, event string, log types.Log, _abi abi.ABI) error {
 	if len(log.Data) > 0 {
 		if err := _abi.Unpack(decodedEvent, event, log.Data); err != nil {
+			if strings.Contains(err.Error(), "Unpack(no-values unmarshalled") {
```

Comment:
@albrow I needed to place this in a slightly different place than what you suggested. This is actually where the event seems to be emitted. 

What's weird is that it seems like the event only has two topics, which suggests that there could be a bug in the way that Geth handles non-indexed topics.